package.json

```diff
@@ -35,10 +35,12 @@
     },
     "workspaces": [
         "packages\\azure-kusto-data",
-        "packages\\azure-kusto-ingest"
+        "packages\\azure-kusto-ingest",
+        "packages\\quick_start"
     ],
     "dependencies": {
         "@lerna-lite/cli": "^1.0.5",
-        "@lerna-lite/run": "^1.0.5"
+        "@lerna-lite/run": "^1.0.5",
+        "@types/node": "^17.0.29"
```

Comment:
Why is this here and not inside of your package?

Reply:
Are you referring to types/node? the other were there originally, this was added to limit the needed node version

Reply:
I think it needs to go to devDependencies, and we are running on node 14, so it shouldn't be more than that